ProcessLib/SourceTerms/CreateNodalSourceTerm.h

```diff
@@ -34,8 +34,8 @@ namespace ProcessLib
 {
 std::unique_ptr<SourceTerm> createNodalSourceTerm(
     BaseLib::ConfigTree const& config, MeshLib::Mesh const& st_mesh,
-    const NumLib::LocalToGlobalIndexMap& dof_table, std::size_t mesh_id,
-    const int variable_id, const int component_id,
+    std::unique_ptr<NumLib::LocalToGlobalIndexMap>&& dof_table,
```

Comment:
[discussion] I know, we have it in several other places done the same way, but I think that `&&` is not needed for a `unique_ptr` because it is move only anyway. Passing it by value is sufficient.

Reply:
Yes, this works. I changed it in nodal, volumetric and the Python source term implementations and rebased.

Reply:
IMHO the rule "Add `&&` to a type if you don't allow copies." is simpler "Add `&&` to a type if you don't allow copies. But if the type is move-only, you can skip the `&&`."